runner/src/server/plugins/engine/plugin.ts

```diff
@@ -88,27 +90,43 @@ export const plugin = {
             basePath: id,
           });
           return h.response({}).code(204);
-        },
-      });
+        } else {
+          consoleMessages.runnerUnavailableAtRoute(request.path);
+          throw Boom.notFound(
+            `previewMode is disabled. To enable it, please switch to development mode 
+by setting NODE_ENV=development in your environment`
+          );
+        }
+      },
+    });
 
-      server.route({
-        method: "get",
-        path: "/published/{id}",
-        handler: (request: HapiRequest, h: HapiResponseToolkit) => {
+    server.route({
+      method: "get",
+      path: "/published/{id}",
+      handler: (request: HapiRequest, h: HapiResponseToolkit) => {
+        if (previewMode) {
           const { id } = request.params;
           if (forms[id]) {
             const { values } = forms[id];
             return h.response(JSON.stringify({ id, values })).code(200);
           } else {
             return h.response({}).code(204);
           }
-        },
-      });
+        } else {
+          consoleMessages.runnerUnavailableAtRoute(request.path);
+          throw Boom.notFound(
```

Comment:
It might be better to document the routes like this `runner/src/server/plugins/initialiseSession/initialiseSession.ts`. When the runner server starts up, it will output some messaging on the routes that have been initialised.

You can see `options: { description: "..." } `. you can use template strings here too

Reply:
Added descriptions to the options property, is that what you meant?